profiles/cli.py

```diff
@@ -20,6 +20,26 @@ def __call__(self, app=None, *args, **kwargs) -> None:
         with app.app_context():
             return self.run(*args, **kwargs)
 
+"Allows calling a method on the Config object to print its result"
+class ReadConfiguration(Command):
+    NAME = 'read-configuration'
+
+    def __init__(self, config) -> None:
+        super(ReadConfiguration, self).__init__()
+        self.config = config
+
+    def get_options(self) -> List[Option]:
+        return [
+            Option('-s', '--method', dest='method', type=str),
+        ]
+
+    # pylint: disable=method-hidden
+    def run(self, method) -> None:
```

Comment:
You will need to add:
https://github.com/elifesciences/profiles/blob/fe32751d4769f76663207e0bb0fba5f34b2da34b/profiles/cli.py#L77

to avoid the same `pylint` issue I had with the last command added.